packages/@okta/vuepress-site/docs/guides/step-up-authentication/main/index.md

```diff
@@ -140,18 +143,19 @@ To check the returned ID token payload, copy the values and paste them into any
 ```json
 {
   "ver": 1,
-  "jti": "AT.tHCWZmHzxS27SqCNpcJAOTTO8UWsKCPXiDGe8J74HzI",
-  "iss": "https://{yourOktadomain}/oauth2/default",
+  "jti": "AT.NovJtQ_NrJ6cgy3h1-638ArovwYXWslu0teQ2M3Ux9c",
+  "iss": "https://sharper.trexcloud.com/oauth2/default",
   "aud": "api://default",
-  "iat": 1660084990,
-  "exp": 1660088590,
+  "iat": 1661289624,
+  "exp": 1661293224,
+  "acr": "urn:okta:loa:1fa:any",
   "cid": "0oa48e74ox4t7mQJX0g7",
   "uid": "00u47ijy7sRLaeSdC0g7",
   "scp": [
     "openid"
   ],
-  "auth_time": 1660084953,
-  "sub": "joe.smith@example.com"
+  "auth_time": 1661289603,
+  "sub": "susan.harper@okta.com"
```

Comment:
Use some fake email like `foo.bar@example.com`

Reply:
yep yep - in my haste to get the examples updated, I forgot to fix this stuff. thank you